partcad-ide-vscode/resources/js/inspector.js

```
@@ -0,0 +1,408 @@
+            if (config["parameters"]) {
+              html += `<tr><td colspan=2>Parameters:</td></tr>`;
+              for (const paramName in config["parameters"]) {
+                const param = config["parameters"][paramName];
+                if (param["type"] === "array") {
+                  /* FIXME(clairbee): Not implemented in VS Code yet */
+                  continue;
+                }
+
+                html += `<tr><td>${paramName}:</td><td>`;
+                let value = '';
+                if (message.params && message.params[paramName]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)